packages/apollo-cli/src/commands/schema/download.ts

```diff
@@ -42,29 +48,56 @@ export default class SchemaDownload extends Command {
       required: true,
       default: "schema.json"
     }
-  ]
+  ];
 
   async run() {
     const { flags, args } = this.parse(SchemaDownload);
 
     const header = Array.isArray(flags.header) ? flags.header : [flags.header];
 
-    const apiKey = flags.key;
-    const pullFromEngine = !!apiKey && !flags.endpoint;
-
     const tasks: Listr = new Listr([
-      loadSchemaStep(this, pullFromEngine, apiKey, flags.engine, "Fetching local schema", async (ctx) => {
-        ctx.schema = await fetchSchema({
-          endpoint: flags.endpoint,
-          header: header.filter(x => Boolean(x)).map(x => JSON.parse(x)),
-        })
-      }),
       {
-        title: `Saving schema to ${args.output}`,
+        title: "Loading Apollo config",
         task: async ctx => {
-          await promisify(fs.writeFile)(args.output, JSON.stringify(ctx.schema));
-        },
+          if (flags.config) {
+            ctx.config = loadConfigFromFile(flags.config) || {};
```

Comment:
it feels like we should be able to combine this and the next function for a shared `loadConfig(path?)` option that defaults to process.cwd()